docs/functional/cart.md

```diff
@@ -0,0 +1,29 @@
+# Cart
```

Comment:
I miss in the article the information about the notification when product price is changed or is not available any more.

Reply:
The information about notification is added.